lib/features/diary_detail/diary_detail_screen.dart

```diff
@@ -0,0 +1,89 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_quill/flutter_quill.dart';
+import 'package:interactive_diary/features/diary_detail/bloc/diary_detail_cubit.dart';
+import 'package:interactive_diary/features/diary_detail/widgets/diary_header.dart';
+import 'package:nartus_ui_package/dimens/dimens.dart';
+import 'package:nartus_ui_package/theme/nartus_theme.dart';
+import 'package:nartus_ui_package/widgets/buttons/nartus_button.dart';
+import 'package:interactive_diary/gen/assets.gen.dart';
+import 'package:interactive_diary/generated/l10n.dart';
+
+class DiaryDetailScreen extends StatelessWidget {
+  const DiaryDetailScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => BlocProvider<DiaryDetailCubit>(
+        create: (context) => DiaryDetailCubit()..loadDiaryDetail(),
+        child: const DiaryDetailBody(),
+      );
+}
+
+class DiaryDetailBody extends StatefulWidget {
+  const DiaryDetailBody({Key? key}) : super(key: key);
+
+  @override
+  State<DiaryDetailBody> createState() => _DiaryDetailBodyState();
+}
+
+class _DiaryDetailBodyState extends State<DiaryDetailBody> {
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<DiaryDetailCubit, DiaryDetailState>(
```

Comment:
BlocConsumer is not needed here, we do not need any action listener yet. When the future comes, we'll add, for now, do not use it.